docker-compose-base.yaml

```diff
@@ -10,13 +10,14 @@ services:
       test: >
         curl -f -X POST http://localhost:8080/graphql
         -H "Content-Type: application/json"
-        --data '{ "query": "{ productItems{totalCount} }" }' || exit 1
+        --data '{ "query": "{ healthcheck }" }' || exit 1
```

Comment:
I'm not sure if we want to have this in the GraphQL schema, definitely not if it is not ommited from the supergraph schema. Health is sth internal which we don't want to expos to our users.
Either way, I would recommend you to talk with @delvh  about this. I remember he wanted me to add a health endpoint in a more or less k8s best practices way. Ideally, we could build this more or less identical across all our services

Reply:
AFAIK, typical best practices are offering a `/health[z]` and/or `/live[z]`/`/ready[z]` endpoint.
In general, I think it is a much better idea to approach it that way since then the health check is dependency free, which is a requirement for a healthcheck.
What good is a health check if it cannot be called?
I don't quite care what the actual endpoint is, as long as we decide on one.

Reply:
let's discuss how we want to structure our health endpoint in Monday during the daily